pkgs/development/python-modules/block-io/default.nix

```diff
@@ -12,7 +12,13 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     base58
     ecdsa
-    pycryptodome
+    (pycryptodome.overrideAttrs (x: {
+      src = fetchPypi {
+        pname = "pycryptodome";
+        version = "3.6.6";
+        sha256 = "b3cb4af317d9b84f6df50f0cfa6840ba69556af637a83fd971537823e13d601a";
+      };
+    }))
```

Comment:
We can't do that for Python packages. You should instead patch the source.

Reply:
you mean make it work with the latest version of `pycryptodome` ?

Reply:
Exactly. I guess the only thing that's necessary for that is patching setup.py.

Reply:
I tested it with the last pycryptodome version and opened a PR upstream, this PR is updated with the corresponding patch